optacloud/src/main/java/optacloud/optacloud/Process.java

```diff
@@ -0,0 +1,76 @@
+package optacloud.optacloud;
+
+/**
+ * This class was automatically generated by the data modeler tool.
+ */
+
+@org.optaplanner.core.api.domain.entity.PlanningEntity
+public class Process implements java.io.Serializable
+{
+
+   static final long serialVersionUID = 1L;
+
+   @org.kie.api.definition.type.Label(value = "Required CPU power")
+   private int requiredCpuPower;
+   @org.kie.api.definition.type.Label(value = "Required memory")
+   private int requiredMemory;
+   @org.kie.api.definition.type.Label(value = "Required network bandwidth")
+   private int requiredNetworkBandwidth;
+   @org.kie.api.definition.type.Label(value = "Computer")
+   @org.optaplanner.core.api.domain.variable.PlanningVariable(valueRangeProviderRefs = { "computerRange" })
+   private optacloud.optacloud.Computer computer;
+
+   public Process()
+   {
+   }
+
+   public int getRequiredCpuPower()
+   {
```

Comment:
The workbench generates the left bracket on a new line. It's worth discussing whether the style of the generated output should be adjusted (+1 for starting on the same line as method name declaration).